sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -207,8 +205,8 @@ def get_transceiver_bulk_status(self):
         for i in range(1, self.NUM_CHANNELS + 1):
             bulk_status["tx%ddisable" % i] = tx_disable[i-1] if self.get_tx_disable_support() else 'N/A'
             bulk_status["tx%dbias" % i] = tx_bias[i - 1]
-            bulk_status["rx%dpower" % i] = float("{:.3f}".format(self.mw_to_dbm(rx_power[i - 1]))) if self.get_rx_power_support() else 'N/A'
-            bulk_status["tx%dpower" % i] = float("{:.3f}".format(self.mw_to_dbm(tx_power[i - 1]))) if self.get_tx_power_support() else 'N/A'
+            bulk_status["rx%dpower" % i] = float("{:.3f}".format(self.mw_to_dbm(rx_power[i - 1]))) if self.get_rx_power_support() and rx_power[i - 1] != 'N/A' else 'N/A'
```

Comment:
why would rx_power be 'N/A'. If its due to due to i2c failure then that should be fixed because we have so many fields to fix. What about tx_disable[]?

Reply:
@stephenxs How do I distinguish the RX power = 'N/A' is because of bad optics or due to the optics itself does not support?

Reply:
If it is caused by bad optics, there will be other errors from drivers. So we do not distinguish it here.
How do you think?

Reply:
@stephenxs get_rx_power() will return 'None' so that also needs to be taken care of here

Reply:
`get_rx_power()` returns `None` only if `self.get_rx_power_support()` is `False`. We handle that case by checking `self.get_rx_power_support()`, right?
Same as `tx`.

Reply:
get_rx_power() can return None on i2c read failure too. could you check